src/OctoshiftCLI.Tests/AdoClientTests.cs

```diff
@@ -188,17 +253,22 @@ public async Task GetAsync_Logs_The_Response_Status_Code_And_Content()
         public async Task GetAsync_Throws_HttpRequestException_On_Non_Success_Response()
         {
             // Arrange
-            using var httpResponse = new HttpResponseMessage(HttpStatusCode.InternalServerError);
-            var handlerMock = MockHttpHandler(_ => true, httpResponse);
+            var httpResponse = () => new HttpResponseMessage(HttpStatusCode.InternalServerError);
```

Comment:
So this is basically a way to cheat and suppress the disposable warning, my suggestion is to use the previous code but suppress the warning for this line only, because it's been used in a unit test 😄

Reply:
it's a bit more than that, the previous code would only construct 1x HttpResponseMessage, the lambda code constructs multiple (1 per call).  And that's necessary because somewhere in the stack it's disposing the HttpResponseMessage after each call.